fullstop-violation/fullstop-violation-jpa/src/generated/queryDsl/org/zalando/stups/fullstop/violation/domain/QAbstractCreatableEntity.java

```diff
@@ -0,0 +1,44 @@
+package org.zalando.stups.fullstop.violation.domain;
+
+import static com.mysema.query.types.PathMetadataFactory.*;
+
+import com.mysema.query.types.path.*;
+
+import com.mysema.query.types.PathMetadata;
+import javax.annotation.Generated;
+import com.mysema.query.types.Path;
+
+
+/**
+ * QAbstractCreatableEntity is a Querydsl query type for AbstractCreatableEntity
+ */
+@Generated("com.mysema.query.codegen.SupertypeSerializer")
+public class QAbstractCreatableEntity extends EntityPathBase<AbstractCreatableEntity> {
+
```

Comment:
You are absolutely right. Just saw that the are in 'src/generated/....' instead of 'target/generated-classes/queryDsl' did someone this configuration for a special reason?